CHANGELOG.md

```diff
@@ -2,6 +2,99 @@
 
 ## 5.14.8
 
+<!-- generated comparing v5.14.8..master -->
+
+_Sep 11, 2023_
+
+A big thanks to the 15 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎉 Added `Drawer` component to Joy UI (#38169) @mnajdova
```

Comment:
A link to the docs could be great.

Reply:
Added link to the docs
@siriwatknp @zanivan, let me know if you have an image or video you wish to add 😊

Reply:
I don't agree here about images or video clips. We can brag about new features on social media, but the changelog should be concise and highlight changes from the previous version. It's not a means of marketing.

A link is fine, though.

Reply:
Yep, link is enough for the changelog.

Reply:
> I don't agree here about images or video clips. We can brag about new features on social media, but the changelog should be concise and highlight changes from the previous version. It's not a means of marketing.

@michaldudak There are different schools on this.

1. Changelog first: https://github.com/prisma/prisma/releases, https://github.com/drizzle-team/drizzle-orm/releases, 
2. Twitter first: https://github.com/chakra-ui/chakra-ui/releases https://github.com/shadcn-ui/ui/releases
3. Blog posts first: https://github.com/microsoft/TypeScript/releases, https://nextjs.org/blog, https://v5.mantine.dev/changelog/5-8-0/
4. Doesn't announce

I feel the problem in our case is that we fall most often under "Doesn't announce" 😁. I would be much in favor of 3.